ui/app/pages/settings/index.scss

```diff
@@ -205,10 +205,6 @@
     }
   }
 
-  &__copyable-address {
```

Comment:
:laughing: 
Funny that that class isn't even used by the `Copyable` component! I thought it was too though - I was planning to ask you to delete this if you'd forgotten.

Though it _is_ unused, so it should still be deleted.

Reply:
It's unused and it has copyable in the name, how could I leave it 🔥